guides/source/routing.md

```diff
@@ -1136,19 +1136,19 @@ For example, here's a small section of the `rails routes` output for a RESTful r
 edit_user GET    /users/:id/edit(.:format) users#edit
 ```
 
-You can search through your routes with the --grep option (-g for short). This outputs any routes that partially match the URL helper method name, the HTTP verb, or the URL path.
+You can search through your routes with the -g option. This outputs any routes that partially match the URL helper method name, the HTTP verb, or the URL path.
```

Comment:
Let's say `the grep option: -g`. Makes it clearer why the option has that name if we make it googlable :)